### DISCUSSION TITLE
How can I provide different vite config for each entrypoint ?

### DISCUSSION THREAD

**Discussion by laizp** [wxt-dev/wxt#1254] [Q&A] [answered]
I want to know which entrypoint is using current vite config, so I can change some options of vite config for a specific entrypoint.

**Comment by aiktb** [comment, marked as answer]
see this https://wxt.dev/api/reference/wxt/interfaces/WxtHooks.html#vite-build-extendconfig

https://wxt.dev/api/reference/wxt/interfaces/WxtHooks.html#vite-devserver-extendconfig

It seems that hook in dev mode can only add new vite config to all entrypoints

**Reply by laizp** [reply]
Thank you. It solves my problem. I just want to use different vite config for entrypoints at build stage.
